samples/bluetooth/throughput/README.rst

```diff
@@ -24,6 +24,9 @@ ATT_MTU size
    In *Bluetooth* Low Energy, the default Maximum Transmission Unit (MTU) is 23 bytes.
    When increasing this value, longer ATT payloads can be achieved, increasing ATT throughput.
 
+.. note::
+   You can configure the ATT_MTU size only using menuconfig and then compiling and programming the sample again.
```

Comment:
Proposing: To configure the ATT_MTU size, use the ``menuconfig`` and compile and program the sample again.
Depending on what you mean. This is a bit ambiguous as now.
Question: Does this mean that the only way to configure the ATT_MTU size is to use menuconfig and...
OR To configure the ATT_MTU only, one should use menuconfig and...

Reply:
This means that the only way to configure the ATT_MTU size is....

Reply:
I chose this sentence: "To configure the ATT_MTU size, use menuconfig and compile and program the sample again."